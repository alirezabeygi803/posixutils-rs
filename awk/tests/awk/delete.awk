BEGIN {
    a["test"] = "test";
    a["other"] = "other";
    print a["test"];
    print a["other"];
    delete a["test"];
    print a["test"];
    print a["other"];

    b[0] = 1;
    b[1] = 2;
    b[2] = 3;
    b[3] = 4;
    print b[0];
    print b[1];
    print b[2];
    print b[3];
    delete b;
    print b[0];
    print b[1];
    print b[2];
    print b[3];
}
