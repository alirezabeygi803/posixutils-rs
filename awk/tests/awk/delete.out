test
other

other
1
2
3
4




